TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /scan
  QUEUE_SIZE: 1
  LEFT_DOWN: [-0.25, 0]
  RIGHT_UP: [0.25,0.25]

XYCAR:
  START_SPEED: 20.0
  MAX_SPEED: 30.0
  MIN_SPEED: 10.0
  SPEED_CONTROL_THRESHOLD: 15.0
  ACCELERATION_STEP: 0.1
  DECELERATION_STEP: 2.0
  ACCELERATION_STEP_S: 0.1
  DECELERATION_STEP_S: 0.4